examples/hybrid/topography_helper.jl

```diff
@@ -0,0 +1,49 @@
+function topo_elev_dataset_path()
+    etopo1_elev_data = AW.ArtifactWrapper(
+        @__DIR__,
+        "topo-elev-info",
+        AW.ArtifactFile[AW.ArtifactFile(
+            url = "https://caltech.box.com/shared/static/gvilybsu5avxso1wubxjthpip9skc6mf.nc",
+            filename = "ETOPO1_coarse.nc",
+        ),],
+    )
+    return AW.get_data_folder(etopo1_elev_data)
+end
```

Comment:
Should we move this into `artifacts/artifact_funcs.jl` with the other ones?

Reply:
That way we can also delete `include(joinpath(@__DIR__, "../examples/hybrid/topography_helper.jl"))` above

Reply:
The rest of this file could go into `src/topography.jl`

Reply:
Moved